python/examples/querying-a-single-package-more-info.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/python3
+
+# Query a single package but this time print more information.
+# A Python equivalent for `rpm -qi hello`
+
+import rpm
+
+ts = rpm.TransactionSet()
+mi = ts.dbMatch("name", "hello")
+hdr = next(mi, None)
+if hdr:
+    print(
+        f"Name        : {hdr[rpm.RPMTAG_NAME]}\n"
+        f"Version     : {hdr[rpm.RPMTAG_VERSION]}\n"
+        f"Release     : {hdr[rpm.RPMTAG_RELEASE]}\n"
+        f"Architecture: {hdr[rpm.RPMTAG_ARCH]}\n"
+        f"Install Date: {hdr[rpm.RPMTAG_INSTALLTIME]}\n"
+        f"Group       : {hdr[rpm.RPMTAG_GROUP]}\n"
+        f"Size        : {hdr[rpm.RPMTAG_SIZE]}\n"
+        f"License     : {hdr[rpm.RPMTAG_LICENSE]}\n"
+        f"Signature   : TODO\n"
```

Comment:
What is the correct tag here, please?

Reply:
The answer is not a single tag, it's either RPMTAG_RSAHEADER or RPMTAG_DSAHEADER (will cover both original DSA and ECDSA)

Most people prefer the shorter hdr['license'] syntax, but of course using rpm.RPMTAG_* symbols, errors are more up front.